### PR TITLE
[Merged by Bors] - feat: inverse of LinearPMap

### PR DESCRIPTION
**Pull request by mcdoll**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/LinearPMap.lean [hunk 1]
```diff
@@ -674,6 +674,20 @@ theorem mem_graph_iff (f : E →ₗ.[R] F) {x : E × F} :
 theorem mem_graph (f : E →ₗ.[R] F) (x : domain f) : ((x : E), f x) ∈ f.graph := by simp
 #align linear_pmap.mem_graph LinearPMap.mem_graph
 
+theorem graph_map_fst_eq_domain (f : E →ₗ.[R] F) : f.graph.map (LinearMap.fst R E F) = f.domain :=
+  by
```
Comment (semorrison) [inline Mathlib/LinearAlgebra/LinearPMap.lean:678]:
```suggestion
theorem graph_map_fst_eq_domain (f : E →ₗ.[R] F) : f.graph.map (LinearMap.fst R E F) = f.domain := by
```

**Comment by semorrison** [comment]
bors d+

File: Mathlib/LinearAlgebra/LinearPMap.lean (hunk 1 above)
Comment (mcdoll) [reply Mathlib/LinearAlgebra/LinearPMap.lean:678]: This gets over the 100 chars limit.

**Comment by mcdoll** [comment]
bors merge

**Pull request closed without merging**